cmd/setup.go

```diff
@@ -95,7 +95,7 @@ func configureEnvironment(cmd *cobra.Command, conf config) (err error) {
 
 	// setup telemetry
 	if err == nil {
-		err = telemetry.Create(conf.Plant)
```

Comment:
Brauchen wir hier den Error nicht trotzdem? Wenn in der Config `telemetry: true` steht muss es die ID doch geben?

Reply:
`create` macht jetzt keine Prüfung auf Machine-/Config-ID mehr. Das wird nun nur noch geprüft wenn man versucht telemetry wirklich zu aktivieren (enable). Dieses Aktivieren kann noch während des Setups sein (siehe folgende Zeilen) oder es kann zur Laufzeit auftreten (via UI).

Im ersten Fall (`telemetry: true`, kein `plant`) startet evcc nicht und wirft einen Fehler der den User auffordert die Config zu erweitern. Das ist genau das Verhalten was wir bislang ja sowieso schon hatten. Hier ist der Fehlertext nun etwas ausführlicher.

Im zweiten Fall (Aktivieren in UI, kein `plant`) bekommt der User die Fehlermeldung in der UI (siehe Screenshot).

Reply:
`create` an sich muss aber bleiben, weil wir ja `plant: ...` irgendwie für spätere Aktivierung speichern müssen und uns ggf. die Machine-ID erzeugen lassen. Daher wird das immer aufgerufen.

Reply:
> Im ersten Fall (telemetry: true, kein plant) startet evcc nicht und wirft einen Fehler der den User auffordert die Config zu erweitern

Wo sehe ich das? Dafür müsste ja `Enable` aufgerufen werden?